src/Entity/Cart.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\CartRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: CartRepository::class)]
+class Cart
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column]
+    private ?bool $status = null;
+
+    #[ORM\Column(type: Types::DATETIME_MUTABLE)]
+    private ?\DateTimeInterface $date = null;
+
+    #[ORM\ManyToOne(inversedBy: 'carts')]
```

Comment:
One to one relation instead ? A user only gets one cart ?

Reply:
le panier a un statut valide ou non validé se qui permet de rajouter des produit si il n'est pas validé.